plugins/entando-plugin-jacms/src/main/java/org/entando/entando/plugins/jacms/web/resource/ResourcesController.java

```diff
@@ -76,20 +79,22 @@
     @ApiResponses(value = {
             @ApiResponse(code = 200, message = "OK"),
             @ApiResponse(code = 401, message = "Unauthorized")})
-    @PostMapping("/plugins/cms/assets")
+    @PostMapping(value = "/plugins/cms/assets"/*, consumes = MediaType.MULTIPART_FORM_DATA_VALUE*/)
```

Comment:
Is that consumes commented out? Remove if so and add a proper comment if one is needed

Reply:
Sorry, was committed accidentally.
Cleaning up...